Sources/Property.swift

```diff
@@ -32,6 +29,16 @@ public protocol PropertyProtocol: class, BindingSource {
 	var signal: Signal<Value, NoError> { get }
 }
 
+/// Represents a property that allows observation of its changes.
+///
+/// Only classes can conform to this protocol, because having a signal
+/// for changes over time implies the origin must have a unique identity.
```

Comment:
I think you need this docstring next to the type so that it gets exposed to clients (and duplicated).